docs/classes/PackDefinitionBuilder.html

```diff
@@ -128,7 +128,7 @@ <h3>constructor</h3>
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/coda/packs-sdk/blob/main/builder.ts#L51">builder.ts:51</a></li>
```

Comment:
Hmm, looks a bit busted, we seem to have lost all of the source code links?

Reply:
🤦 I read the diff backwards when I checked it before. I misunderstood the `gitRemote` option, it needs already existing remote, not a URL. Reverted.